src/lib/components/calling/Participant.svelte

```diff
@@ -83,6 +84,23 @@
         height: fit-content;
         position: relative;
 
+        .simple {
+            display: flex;
+            flex-direction: column;
+            justify-content: center;
+            align-items: center;
+            &:hover {
+                cursor: pointer;
+            }
+        }
+
+        .user-name {
+            color: white;
+            padding-top: 8px;
+            font-size: 14px;
```

Comment:
lets use the text component and use the size={Size.Smaller}  (or smallest) for this so that it matches up